documentation/wiki/product/decisions/adr/2025-03-24-migrate-existing-api-consumers.md

```diff
@@ -0,0 +1,122 @@
+# Migrate Existing API Consumers
+
+- **Status:** Active
+- **Last Modified:** 2025-04-23
+- **Related Issue:** [#4199](https://github.com/HHS/simpler-grants-gov/issues/4199)
+- **Deciders:** Julius, Lucas
+- **Tags:** api, soap, s2s, applicant, grantor
+
+## Context and Problem Statement
+
+We have a large existing S2S API client base split into two primary cohorts:
+
+1. Other Federal Government Agencies - Grantors, USASpending, etc.
+2. Applicants (data suggests this is primarily Universities, Hospital/Healthcare Groups, State/County/Local Governments)
+
+The level of S2S utilization, size of the user community, and the major differences between SOAP and REST and the authentication methods for the existing and Simpler APIs, leads us to the finding that we will need a way to continuing supporting the SOAP dialect of these APIs for the foreseeable future. Given that many API consumers are other Federal Agency IT Systems, we're anticipating needing a long runway, once the Simpler API has feature parity, to allow agencies to budget, contract, develop, and deploy changes to their systems that will consume the Simpler API instead of the SOAP API.
```

Comment:
[nit] `deploy changes to their systems that will consume the Simpler API instead of the SOAP API.` - I'd suggest ` deploy changes to their systems that will consume the Simpler REST API instead of the SOAP API.`, since there will be a time where Simpler is supporting a SOAP API as well